dashboard/controllers/filesController.go

```diff
@@ -22,14 +22,14 @@ const (
 
 func connectAWS() *session.Session {
 
-	fmt.Println("Connecting to AWS", os.Getenv("AWS_ACCESS_KEY_ID"), os.Getenv("AWS_SECRET_ACCESS_KEY"))
+	log.Info("Connecting to AWS", os.Getenv("AWS_ACCESS_KEY_ID"), os.Getenv("AWS_SECRET_ACCESS_KEY"))
```

Comment:
can you remove this part? we should not log important information like AWS keys and secret <3